macmillanhighered/jenkins-dashboard
# LP-5040 AMI Deploy 2.0 Dashboard Optimization

- What does this PR do?
This AMI dashboard displays the deployment details of each components of all the environments.
- Are there critical parts that deserve extra attention?
No
- Where should the reviewer start?
From App and Web.config file
- How should this be manually tested; or how was it tested?
Manual
- Any background context or additional details you want to provide?
No
- Is there any change included on this PR that does not belong to the JIRA ticket?
No
- Screenshots (if appropriate)

Verify that the PR complies with all these checks:
- [ ] Updated documentation (if required)
- [x] Changes maintain the indentation of original files.
- [x] PR title follows the format "LP-XXXX: \<Ticket Title\>" | "LP-XXXX: \<general purpose of the PR\>"
- [ ] PR title is prefixed with "[WIP]" since it is not ready for review.
- [ ] Relevant changes are commented inline for simpler understanding (specially for long PRs)
- [ ] Checked with the accessibilityTester and gulp accessibilityTest task for new issues.
- [ ] New code is on par with latest ECMA features ([Examples of ECMA6]
- [ ] Consider using the vars.scss for reusable style values.
